design-library/src/components/BccRange/BccRange.css

```diff
@@ -0,0 +1,54 @@
+@layer components {
+    .bcc-range {
+        @apply flex flex-col gap-2 p-8 w-full;
+    }
+
+    .bcc-range__container {
+        @apply flex flex-col items-center gap-2 w-full relative;
+    }
+
+    .bcc-range__text {
+        @apply text-sm text-gray-700 whitespace-nowrap;
+    }
+
+    .bcc-range__value {
+        @apply flex absolute bottom-4 -translate-x-1/2 bg-danger mb-2 text-sm rounded-lg w-12 h-8 text-center items-center justify-center shadow-md;
+    }
+
+    .bcc-range__input {
+        @apply w-full h-1 bg-gray-200 rounded appearance-none cursor-pointer;
+        @apply disabled:opacity-50 disabled:cursor-not-allowed;
+        -webkit-appearance: none;
+        appearance: none;
+    }
+
+    .bcc-range__input:disabled {
+        @apply bg-gray-400 cursor-not-allowed;
+    }
+
+    .bcc-range__input::-webkit-slider-thumb {
+        -webkit-appearance: none;
+        @apply w-5 h-5 mt-[-8px] rounded-full bg-white cursor-pointer;
+        box-shadow: 0 1px 2px 0px rgba(113, 113, 113, 0.5),
```

Comment:
If existing shadow presets don't match we should let the designer know so they use existing presets. eg. shadow-lg?